python/py-MechanicalSoup/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        MechanicalSoup MechanicalSoup 1.1.0 v
+revision            0
+name                py-MechanicalSoup
```

Comment:
please don't capitalize portnames - we've been trying to use [`upt`](https://github.com/macports/upt-macports) to make generating ports more easily from several sources. To make that work more consistently the idea there was not to capitalize portnames even if upstream does. It's not consistently done throughout MacPorts yeb, but can you please change this?